package.json

```diff
@@ -36,12 +36,12 @@
     "babel-preset-stage-0": "^6.0.15",
     "eslint": "^1.10.3",
     "eslint-plugin-react": "^3.6.2",
-    "react-hot-loader": "^1.3.0",
+    "react-hot-loader": "^3.0.0-beta.1",
```

Comment:
The distinction between `devDependencies` and `dependencies` is useful when the package is going to be required by another package. Since this project will not be included elsewhere it's up to the developer I'd say.

For web apps you can either interpret production dependencies as
1. the minimal set of packages required to build the application
2. the set of packages a server needs to install before it can run the application, given it already contains the bundled files (emitted through webpack). This happens when doing SSR and excluding certain packages from webpack.

There's advantages to both interpretations. I think it depends on the project itself. I'm curious what others think.

Either way, `react-hot-loader` should be in the same category as  `react` and `react-dom` for the reasons you mentioned.